README.md

```diff
@@ -1,39 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
```

Comment:
- [ ]  [OPTIONAL] Good job working on setting up a 'Awesome books: with ES6' project 😃, however, this note is only an instruction for you to set up the license. I think you should remove it to keep your README more professional. 👍

![Screenshot 2023-05-01 114556](https://user-images.githubusercontent.com/57408419/235436848-5eab4074-ad88-4ab9-8c0d-afb21cce6c90.png)

Reply:
> owww thanks I will remove it.